v2/setup/mesos-master.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+source /etc/environment
+
+HOMEDIR=$(eval echo "~`whoami`")
+
+AWS_CREDS=""
+if [ ! -z $AWS_ACCESS_KEY ]; then
+    AWS_CREDS=" -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY \
+     -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_KEY "
+fi
+
+sudo docker run --rm \
+    -v ${HOMEDIR}:/data/ $AWS_CREDS behance/docker-aws-s3-downloader \
```

Comment:
@bryanlatten because we pull from Behance in numerous places and the change you're proposing would require an update to .dockercfg (new Docker Hub location) as well as all the images that get pulled down elsewhere. Much better to do in a separate PR.